### ISSUE TITLE
Any external link for pre-Trained models

### ISSUE THREAD

**Issue by makamkumar** [spookycouch/lasr_object_detection_yolo#1]
The pretrained models require login password
Can you give a external link  to try it out
Thanks in advance

**Comment by spookycouch** [comment]
Hi, you can actually use the official weights from Redmon et al. to test it out: https://pjreddie.com/darknet/yolo/

You will also need a classes.txt file with all the names of the labels in the coco dataset. I think this link has one if you need it quickly: https://www.pyimagesearch.com/2018/11/12/yolo-object-detection-with-opencv/
